dolphinscheduler-ui/src/views/projects/task/components/node/tasks/use-sql.ts

```diff
@@ -46,6 +46,7 @@ export function useSql({
     timeout: 30,
     type: 'MYSQL',
     displayRows: 10,
+    segmentSeparator: '',
```

Comment:
I think we should set a default value of semicolon to reduce repeated operations of users. What do you think?

Reply:
I don't think so, because this parameter refers to: this data source does not support the execution of multiple sql statements at a time. This parameter needs to be added, so that the program can split the sql segment and perform multiple queries; 

for example, Hive supports one execution When there are multiple SQL statements, you do not need to add this parameter; Mysql does not support executing multiple statements at a time, you can use this new function, write multiple statements in the sql script of SqlTask, add ; to the segmentSeparator

Reply:
Yes. But i think it's too difficult for users to judge whether this parameter needs to be added. Is it better that we help users judge whether this parameter needs to be added instead of letting users to defined? @zhuangchong

Reply:
@SbloodyS 
This suggestion is very good. I think whether this parameter needs to be related to the data source, provide default values ​​for different data sources, but this will change the data source module. I hope to optimize in the next PR, let SQLTask support this function first, and then provide different data sources to change the value of this parameter, what do you think?

Reply:
Sounds goods to me. Thanks @zhuangchong